components/brave_rewards/resources/rewards_panel/lib/interfaces.ts

```diff
@@ -31,7 +32,7 @@ export interface PublisherInfo {
   name: string
   icon: string
   platform: PublisherPlatform | null
-  registered: boolean
+  publisherStatus: PublisherStatus
```

Comment:
Could this be `walletProvider: ExternalWalletProvider | null` instead?

Reply:
As per our Slack convo, we ended up using `supportedWalletProviders: ExternalWalletProvider[]` for this purpose.